### PR TITLE
Bugfix: fix dispatch_results type

### PR DESCRIPTION
**Pull request by wuminzhe**

### CODE REVIEW

File: contracts/periphery/contracts/s2s/types/CommonTypes.sol [hunk 1]
```diff
@@ -197,15 +222,17 @@ library CommonTypes {
             "The data is not enough to decode InboundLaneData"
         );
 
-        uint64 lastConfirmedNonce = decodeUint64(Bytes.substr(_data, compactLength + 49 * length));
+        uint64 lastConfirmedNonce = decodeUint64(
+            Bytes.substr(_data, compactLength + 49 * length)
```
Comment (hujw77) [inline contracts/periphery/contracts/s2s/types/CommonTypes.sol:226]: Why is `49`?

Comment (wuminzhe) [reply contracts/periphery/contracts/s2s/types/CommonTypes.sol:226]: 49 is the bytes length of `UnrewardedRelayer`

Comment (hujw77) [reply contracts/periphery/contracts/s2s/types/CommonTypes.sol:226]:
```
    struct UnrewardedRelayer {
        bytes32 relayer;
          struct DeliveredMessages {
            uint64 begin;
            uint64 end;
            bytes dispatch_results;
        }
    }
```
But it is not a fixed length.

Comment (wuminzhe) [reply contracts/periphery/contracts/s2s/types/CommonTypes.sol:226]:
```
// struct InboundLaneData {
//     VecDeque<UnrewardedRelayer> relayers;
//     uint64 last_confirmed_nonce;
// }
```

Comment (hujw77) [reply contracts/periphery/contracts/s2s/types/CommonTypes.sol:226]: Is it fixed? cc @AsceticBear

Comment (wuminzhe) [reply contracts/periphery/contracts/s2s/types/CommonTypes.sol:226]: > ```
>     struct UnrewardedRelayer {
>         bytes32 relayer;
>           struct DeliveredMessages {
>             uint64 begin;
>             uint64 end;
>             bytes dispatch_results;
>         }
>     }
> ```
> 
> But it is not a fixed length.

you are right. the DeliveredMessages is changed

Comment (wuminzhe) [reply contracts/periphery/contracts/s2s/types/CommonTypes.sol]: pls review again.

**Review by hujw77 — APPROVED** [review]
LGTM

**Review by AsceticBear — APPROVED** [review]

**Pull request merged**